person.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  def iinitialize(name, age = 'unknown', parent_permission = 'true')
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  # getter
+  attr_accessor :name
+  attr_accessor :age
```

Comment:
- [ ] I think it would be better if your attr_accessor is just one line like` attr_accessor :name, :age`, please kindly fix that 😁😁